ftdetect/jsonc.vim

```diff
@@ -0,0 +1,2 @@
+autocmd BufNewFile,BufRead *.jsonc set ft=jsonc
```

Comment:
upstream already has this:
```
runtime/filetype.vim
889:au BufNewFile,BufRead *.jsonc                       setf jsonc
```

Reply:
I didn't know that. I was referring to the following: https://github.com/neoclide/jsonc.vim/blob/master/ftdetect/jsonc.vim